include/bitcoin/blockchain/interface/block_chain.hpp

```diff
@@ -181,8 +181,8 @@ class BCB_API block_chain
     void fetch_block(const hash_digest& hash,
         block_fetch_handler handler) const;
 
-    void fetch_block_txs_size(const hash_digest& hash,
-        block_txs_size_fetch_handler handler) const;
+    void fetch_header_txs_size(const hash_digest& hash,
```

Comment:
`fetch_block_header_txs_size()` should be more appropriate to be consistent with `fetch_block_header()`. Same with the callback type `header_txs_size_fetch_handler`.